types/tunajs/index.d.ts

```diff
@@ -0,0 +1,839 @@
+// Type definitions for tunajs 1.0
+// Project: https://github.com/Theodeus/tuna#readme
+// Definitions by: Yehyoung Kang <https://github.com/pastelmind>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 5.1
```

Comment:
```ts
// Minimum TypeScript Version: 5.1
```
why do you need 5.1 version? (current one). If there are no language specific semantics required, please just drop version from header. It will be tested against:
https://github.com/DefinitelyTyped/DefinitelyTyped#support-window

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e8406c4b983ae3d9e3ad1bdd0d298da5d105e2f6/types/tunajs/index.d.ts#L288-L289

Because of this. The getter and setter uses unrelated types which wans't possible until TypeScript 5.1
Curious to know if there is an alternative that correctly describes the semantics of this field in TS < 5.1

Reply:
no, off there is (was) no other way, thx!